aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -341,6 +341,30 @@ void sync(const c10::List<c10::optional<Tensor>> t_list) {
   }
 }
 
+void replace_(const Tensor& functional_tensor, const Tensor& other) {
+  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(isFunctionalTensor(functional_tensor));
+  unsafeGetFunctionalWrapper(functional_tensor)->replace_(other);
+}
+
+void replace_(const TensorList functional_tensor, TensorList other) {
+  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(functional_tensor.size() == other.size());
+  for (const auto i : c10::irange(functional_tensor.size())) {
+    replace_(functional_tensor[i], other[i]);
+  }
+}
+
+
+void commit_update(const Tensor& functional_tensor) {
+  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(isFunctionalTensor(functional_tensor));
+  unsafeGetFunctionalWrapper(functional_tensor)->commit_update();
+}
+
+void commit_update(const TensorList functional_tensor) {
+  for (const auto i : c10::irange(functional_tensor.size())) {
+    commit_update(functional_tensor[i]);
+  }
+}
```

Comment:
These aren't strictly necessary for this PR, but I need them later (accidentally bundled them in this commit)